### PR TITLE
Historical address

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: business_register/converter/uo_historical.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from django.apps import apps
+
+from business_register.models.company_models import Company
+from data_converter import settings_local
+from data_ocean.converter import Converter
+
+
+class Historical(Converter):
+    LOCAL_FILE_NAME = settings_local.LOCAL_FILE_NAME_UO_ADDRESS
+    LOCAL_FOLDER = settings_local.LOCAL_FOLDER
```
Comment (roman-tiukh) [inline business_register/converter/uo_historical.py:11]: use django interface for access to settings
```suggestion
from django.conf import settings
from data_ocean.converter import Converter


class Historical(Converter):
    LOCAL_FILE_NAME = settings.LOCAL_FILE_NAME_UO_ADDRESS
    LOCAL_FOLDER = settings.LOCAL_FOLDER
```

File: business_register/converter/uo_historical.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        for record in records:
+            company = self.HistoricalCompany()
+            company.edrpou = record.xpath('EDRPOU')[0].text
+            company.address = record.xpath('Address')[0].text
+            company.history_date = datetime.datetime.strptime(
+                record.xpath('DATE')[0].text,
+                "%Y/%m/%d %H:%M:%S"
+            ).strftime("%Y-%m-%d %H:%M:%S")
+            if Company.objects.filter(edrpou=company.edrpou).first():
+                company.id = Company.objects.filter(edrpou=company.edrpou).first().id
```
Comment (roman-tiukh) [inline business_register/converter/uo_historical.py:28]: 2 redundant query to database

File: business_register/converter/uo_historical.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import datetime
+from django.apps import apps
+
+from business_register.models.company_models import Company
+from data_converter import settings_local
+from data_ocean.converter import Converter
+
+
+class Historical(Converter):
+    LOCAL_FILE_NAME = settings_local.LOCAL_FILE_NAME_UO_ADDRESS
```
Comment (roman-tiukh) [inline business_register/converter/uo_historical.py:10]: LOCAL_FILE_NAME_UO_ADDRESS need to be added to settings_local.py

File: business_register/converter/uo_historical.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/converter/uo_historical.py:11]: done

File: business_register/converter/uo_historical.py (hunk 3 above)
Comment (Iva-khar) [reply business_register/converter/uo_historical.py]: done

File: business_register/converter/uo_historical.py (hunk 2 above)
Comment (Iva-khar) [reply business_register/converter/uo_historical.py]: fixed

File: business_register/converter/uo_historical.py (hunk 3 above)
Comment (Iva-khar) [reply business_register/converter/uo_historical.py]: done

File: business_register/converter/uo_historical.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/converter/uo_historical.py]: fixed

File: business_register/converter/uo_historical.py (hunk 2 above)
Comment (Iva-khar) [reply business_register/converter/uo_historical.py]: fixed

**Pull request merged**